SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005993	[Energy Bolt] damage is enhanced to Wizard circle 3 standard.
+SKILL_20151224_005994	[Earthquake] damage is enhanced to Wizard circle 3 standard.
+SKILL_20151224_005995	Deals additional damage equal to 40% magic attack per attribute level when attacking an enemy under [Sleep] with [Energy Bolt].
+SKILL_20151224_005996	Enemies affected by [Lethargy] will receive 20% additional damage with Strike attacks.
+SKILL_20151224_005997	Increases magic damage by 10% per attribute level when [Quick Cast] is active.
+SKILL_20151224_005998	[Enchant Fire] will decrease the fire resistance of nearby enemies by 10 per attribute level for 5 seconds.
+SKILL_20151224_005999	Freeze duration of [Subzero Shield] is increased by 0.5 seconds per attribute level.
+SKILL_20151224_006000	Decreases the physical defense by 10% per attribute level on enemies that are pulled by [Gravity Pole].
+SKILL_20151224_006001	Decreases the AoE Defense Ratio of the enemies gathered through [Hangman's Knot] by 1.
+SKILL_20151224_006002	nemies affected by [Swell Body] will have their movement speed decreased by 15% and physical and magical attack increased by 20% per attribute level.
+SKILL_20151224_006003	Enemies affected by [Shrink Body] will have their movement speed increased by 10% and physical and magical attack decreased by 25% per attribute level.
+SKILL_20151224_006004	When you attack an enemy affected by [Swell Body] while [Left Arm Enhancement] is active, increases the damage dealt on an enemy by 35 per attribute level.
+SKILL_20151224_006005	When you attack an enemy affected by [Swell Body] while [Left Arm Enhancement] is active, it will continuously apply the [Attacked Weakened] status effect. Increases the duration by 8 seconds per attribute level and [Attacked Weakened] can stack up to 10 times.
+SKILL_20151224_006006	Evocation: Enhance
+SKILL_20151224_006007	Increases the damage dealt on an enemy with [Evocation] by 1% per attribute level.
+SKILL_20151224_006008	Desmodus: Enhance
+SKILL_20151224_006009	Increases the damage dealt on an enemy with [Desmodus] by 1% per attribute level.
+SKILL_20151224_006010	Critical hit resistance of the enemy fallen to [Slow] is decreased by 15% of the user's intelligence value.
+SKILL_20151224_006011	Sonic Strike: Enhance
+SKILL_20151224_006012	Increases the damage dealt on an enemy with [Sonic Strike] by 1% per attribute level.
+SKILL_20151224_006013	Increase your accuracy by 5% per attribute level when attacking with gun equipments.
+SKILL_20151224_006014	Wild Shot: Enhance
+SKILL_20151224_006015	Increases the damage dealt on an enemy with [Wild Shot] by 1% per attribute level.
+SKILL_20151224_006016	Cannon Mastery: Penetration
+SKILL_20151224_006017	Increases your block penetration by 6% per attribute level when attacking with Cannoneer skills.
+SKILL_20151224_006018	Airborne enemy hit by [Showdown] is grounded for 1.8 seconds per attribute level.
```

Comment:
Airborne enemies hit by [Showdown] will fall to the ground for 1.8 seconds per attribute level.